parts/k8s/containeraddons/kubernetesmasteraddons-azure-npm-daemonset.yaml

```diff
@@ -79,6 +79,11 @@ spec:
           image: {{ContainerImage "azure-npm-daemonset"}}
           securityContext:
             privileged: true
+          resources:
+            requests:
+              memory: 20Mi

```

Comment:
Let's make this configurable, and set "20mi" as the default. In the yaml spec here you would put:

`{{ContainerMemReqs "azure-npm-daemonset"}}` in place of the static `20mi`

Reply:
fixed